src/material-experimental/mdc-snack-bar/_snack-bar-theme.scss

```diff
@@ -36,10 +36,15 @@
   $mdc-snackbar-label-ink-color: $orig-mdc-snackbar-label-ink-color !global;
   $mdc-snackbar-dismiss-ink-color: $orig-mdc-snackbar-dismiss-ink-color !global;
 
-  .mat-mdc-simple-snack-bar .mdc-snackbar__action {
+  .mat-mdc-simple-snack-bar .mat-mdc-snack-bar-actions .mat-mdc-snack-bar-action {
     $is-dark-theme: map.get($config, is-dark);
     $accent: map.get($config, accent);
     color: if($is-dark-theme, inherit, theming.color($accent, text));
+
+    .mat-ripple-element {
+      background-color: currentColor;
+      opacity: 0.01;
```

Comment:
I copied this over from the original button theme. Is there a variable for the ripple opacity I can use?